oci/manager.go

```diff
@@ -492,19 +492,45 @@ func (manager *ociManager) Catalog(ctx context.Context, qopts ...packmanager.Que
 		}
 	}
 
-searchRemoteIndexes:
 	if query.Remote() {
 		more, err := manager.update(ctx, auths)
 		if err != nil {
 			log.G(ctx).
 				Debugf("could not update: %v", err)
 		} else {
 			for checksum, pack := range more {
+				fullref := pack.Name()
+
+				if qglob != nil && !qglob.Match(fullref) {
+					log.G(ctx).
+						WithField("want", qname).
+						WithField("got", fullref).
+						Trace("skipping manifest: glob does not match")
+					continue
+				} else if qglob == nil {
+					if len(qversion) > 0 && len(qname) > 0 {
+						if fullref != fmt.Sprintf("%s:%s", qname, qversion) {
+							log.G(ctx).
+								WithField("want", fmt.Sprintf("%s:%s", qname, qversion)).
+								WithField("got", fullref).
+								Trace("skipping manifest: name does not match")
```

Comment:
`Trace("skipping manifest: name and version do not match")`

to differentiate from the one below which has the same message